deploy/infrabox/values.yaml

```diff
@@ -31,6 +33,17 @@ image:
     password: #<required if use private_repo>
 
 
+ha:
+    enabled: false
+
+    check_interval: 10
+
+    active_timeout: 60
+
+    entry_host: #<required if use ha>
```

Comment:
name is not consistent. here it's called `entry_host` and in the helm `ha_root_url`.
Maybe we should use `root_url` as global entry point and add `cluster.root_url`

Reply:
Ok, I will make the name consistent